meta-balena-common/recipes-bsp/grub/grub-efi_%.bbappend

```diff
@@ -9,5 +20,19 @@ do_deploy:append:class-target() {
     install -d ${DEPLOYDIR}/grub/${GRUB_TARGET}-efi/
 }
 
-# build in additional required modules
-GRUB_BUILDIN:append = " regexp probe gzio"
+do_mkimage() {
+    cd ${B}
+
+    if [ -f "${DEPLOYDIR}/grub.gpg" ]; then
+        GRUB_PUBKEY_ARG="--pubkey ${DEPLOYDIR}/grub.gpg"
```

Comment:
The two lines above should use `${DEPLOY_DIR_IMAGE}/balena-keys/grub.gpg` rather than `${DEPLOYDIR}/grub.gpg`

Reply:
Not sure I follow this @mtoman. `balena-keys` is a dependency so the keys will also be in `DEPLOYDIR`, won't them?